src/main/resources/db/migration/V20231003__Create_initial_entity_tables.sql

```diff
@@ -658,6 +658,21 @@ CREATE TABLE `announcements`
   `local_site_id` BIGINT                                    NOT NULL,
   `created_at`    TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) NOT NULL,
   `updated_at`    TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) NOT NULL ON UPDATE CURRENT_TIMESTAMP(3)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb4
+  COLLATE = utf8mb4_unicode_ci;
+
+/**
+  Captcha
+ */
+CREATE TABLE `captcha`
+(
+  `id`            BIGINT AUTO_INCREMENT PRIMARY KEY,
+  `uuid`          VARCHAR(36)                               NOT NULL,
```

Comment:
This should be a Unique index as it should 100% be unique ( and a index? @jgrim ) and with it it would be clearer.

Add it under the create table as you can see in the upper part.

Reply:
Agreed